packages/keystone/lib/List/index.js

```diff
@@ -0,0 +1,1287 @@
+/* eslint-disable no-shadow */
```

Comment:
What's this file? I don't see a file that this is replacing?

Reply:
Good question, should definitely not be here in this branch!